### PR TITLE
Feature/fix2

### PR DESCRIPTION
**Pull request by Akiyoshi-Maeda**

### CODE REVIEW

File: timely_release_crawler.rb [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    time_css = '#main-list-table tr:nth-child(' + index.to_s + ') .kjTime'
+    code_css = '#main-list-table tr:nth-child(' + index.to_s + ') .kjCode'
+    name_css = '#main-list-table tr:nth-child(' + index.to_s + ') .kjName'
+    news_item = doc.at_css(news_css)
+    time_item = doc.at_css(time_css)
+    code_item = doc.at_css(code_css)
+    name_item = doc.at_css(name_css)
+    if (news_item == nil || time_item == nil || code_item == nil || name_item == nil) then
+        next
+    end
```
Comment (zakuro112) [inline timely_release_crawler.rb]: test

Comment (zakuro112) [inline timely_release_crawler.rb]: test

Comment (Akiyoshi-Maeda) [inline timely_release_crawler.rb]: replay

Comment (Akiyoshi-Maeda) [inline timely_release_crawler.rb]: reply

File: timely_release_crawler.rb [hunk 2]
```diff
@@ -0,0 +1,105 @@
+require 'rubygems'
+require 'nokogiri'
+require 'open-uri'
+require 'mysql2'
+require 'digest/md5'
```
Comment (Akiyoshi-Maeda) [inline timely_release_crawler.rb]: comment test

File: timely_release_crawler.rb [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    if newsItem == nil then
+        next
+    end
+    text = newsItem.text
+
+    is_important = false
+    word_masters.each do |word_master|
+        is_important = text.include?(word_master['word'])
+        if (is_important != false) then
+            break
```
Comment (zakuro112) [inline timely_release_crawler.rb]: test

**Pull request closed without merging**